ansible_collections/arista/avd/roles/eos_designs/defaults/main.yml

```diff
@@ -204,3 +112,81 @@ connected_endpoints_keys:
 #custom_structured_configuration_prefix allows the user to customize the prefix for Custom Structured Configuration variables.
 custom_structured_configuration_prefix: custom_structured_configuration_
 custom_structured_configuration_list_merge: 'replace'
+
+switch_type_keys:
+  spine:
+    type: spine
+    default_evpn_role: server
+  l3leaf:
+    type: l3leaf
+    connected_endpoints: true
+    default_evpn_role: client
+    mlag_support: true
+    network_services:
+      l2: true
+      l3: true
+    vtep: true
+  l2leaf:
+    type: l2leaf
+    connected_endpoints: true
+    mlag_support: true
+    network_services:
+      l2: true
+    underlay_router: false
+    uplink_type: port-channel
+  super_spine:
+    type: super-spine
+  overlay_controller:
+    type: overlay-controller
+  spline:
```

Comment:
should spline be `default_evpn_role: client` ?

Reply:
It depends a lot on what we define as a spline. If you add those two, it is almost the same as `l3leaf`. We can remove it again until we develop the designs where it should be used.

Reply:
I agree, and we can always turn the knobs on when required!